server/common/src/main/java/org/eclipse/lsp/cobol/common/utils/RangeUtils.java

```diff
@@ -157,6 +157,6 @@ public Range moveByLine(Range range, int count) {
    * @return character size
    */
   public int charSize(Range range) {
-    return range.getEnd().getCharacter() - range.getStart().getCharacter() + 1;
+    return range.getEnd().getCharacter() - range.getStart().getCharacter();
```

Comment:
looks unrelated and may should be get pulled in earlier?

Reply:
Thanks [GitMensch](https://github.com/GitMensch) for reviewing the PR.
Yes you are correct. I have cleaned this PR a little, for reviewing (still a draft).

Reply:
Thanks [GitMensch](https://github.com/GitMensch) for reviewing the PR.
Yes you are correct. I have cleaned this PR a little, for reviewing (still a draft).